src/app/parcels/add/formSections/VoucherNumberCard.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import { CardProps, onChangeText } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+
+const FullNameCard: React.FC<CardProps> = ({ errorSetter, fieldSetter }) => {
+    return (
+        <GenericFormCard
+            title="Voucher Number"
+            required={false}
+            text="This is usually found in the following format: H-00001-00001. If you don't know the voucher number, leave this section blank."
+        >
+            <FreeFormTextInput
+                label="Number"
```

Comment:
The label of different box should be more consistent
The others all have the format `Enter xxxx here` and this one is just `Number`
I actually prefer just `Number` and `Date` etc because it is how the client/add page is formatted (so maybe be consistent with the existing form!)